app/app.go

```diff
@@ -502,6 +502,7 @@ func New(
 		&app.IBCTransferKeeper,
 		&app.bech32IbcKeeper,
 	)
+	app.GravityKeeper.BurnValset(true)
```

Comment:
This is not necessary, it's already set to true by default when creating the keeper

Reply:
let's keep it for clarity